ClientSide(FrontEnd)/clay_/src/App.js

```diff
@@ -0,0 +1,22 @@
+import { useState } from 'react';
+import './App.css';
+import Forgot from './components/forgot';
+import Images from './components/Images';
+import Login from './components/login';
+import Signup from './components/signup';
+import Footer from './components/footer'
+
+function App() {
+  return (
+    <div className="App">
+      {/* <Signup/> */}
```

Comment:
remove the commented code

Reply:
i removed all commented code from current branch